pallets/asset-manager/Cargo.toml

```diff
@@ -1,52 +1,56 @@
 [package]
-authors = ['Manta Network']
+authors = ["Manta Network"]
 edition = "2021"
-homepage = 'https://manta.network'
-license = 'GPL-3.0'
+homepage = "https://manta.network"
+license = "GPL-3.0"
 name = "pallet-asset-manager"
 repository = 'https://github.com/Manta-Network/Manta/'
 version = "3.4.0"
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = '3.1.2', default-features = false }
-frame-benchmarking = { git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.26", default-features = false, optional = true }
+codec = { package = "parity-scale-codec", version = "3.1.2", default-features = false }
+frame-benchmarking = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false, optional = true }
 frame-support = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
 frame-system = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
 log = { version = "0.4.0", default-features = false }
 manta-primitives = { path = "../../primitives/manta", default-features = false }
+# manta-util = { git = "https://github.com/manta-network/manta-rs", branch = "feat/new-circuits", default-features = false }
+pallet-assets = { git = "https://github.com/paritytech/substrate.git", default-features = false, branch = "polkadot-v0.9.26" }
 scale-info = { version = "2.1.2", default-features = false, features = ["derive"] }
+serde = { version = "1.0.140", default-features = false }
 sp-runtime = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
 sp-std = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
-xcm = { git = 'https://github.com/paritytech/polkadot.git', default-features = false, branch = "release-v0.9.26" }
+xcm = { git = "https://github.com/paritytech/polkadot.git", default-features = false, branch = "release-v0.9.26" }
 
 # 3rd party dependencies
 orml-traits = { git = "https://github.com/manta-network/open-runtime-module-library.git", default-features = false, branch = "polkadot-v0.9.26" }
 
 [dev-dependencies]
-pallet-assets = { git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.26" }
-pallet-balances = { git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.26" }
+pallet-assets = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26" }
+pallet-balances = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26" }
 sp-core = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26" }
 sp-io = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26" }
 
 [features]
 default = ["std"]
 runtime-benchmarks = [
```

Comment:
For what I saw, `pallet-assets` is optional for this pallet, only for benchmakring, so it's unnecessary to built into runtime. You can mark `pallet-assets` as `optional = true`, then change this feature like 
```toml
runtime-benchmarks = [
  "dep:frame-benchmarking",
  "frame-support/runtime-benchmarks",
  "frame-system/runtime-benchmarks",
  "manta-primitives/runtime-benchmarks",
  "pallet-assets?/runtime-benchmarks",
]
```
Now when compile the pallet, the count of crates needed to be compiled(`cargo b`) will be reduced from `356` to `336`.
See this cargo feature: https://blog.rust-lang.org/2022/04/07/Rust-1.60.0.html#new-syntax-for-cargo-features

But we have to bump `rust-version`.

Reply:
Actually it's not even needed for benchmarking so I'll remove it altogether.